apps/src/javalab/Neighborhood.js

```diff
@@ -111,4 +114,12 @@ export default class Neighborhood {
     // return 2 to the power of that scaled value to get a multiplier between 0.5 and 2.
     return Math.pow(2, -2 * this.speedSlider.getValue() + 1);
   }
+
+  // Convert y-coordinate from Neighborhood format to Maze format.
+  // In neighborhood (0,0) is the bottom-left grid square, in Maze
+  // it is the top left.
+  convertYCoordinate(y) {
```

Comment:
Will we also need to do this for paint glomming? i.e. I _think_ we pass coordinates for painting?

Reply:
I think that's right, @hannahbergam to confirm

Reply:
Paint glomming is currently set up from top left, yes. I can configure this part now, as I am setting up paint() and scrapePaint().